src/Aquarius.Client/TimeSeries/ExistingSessionAuthenticator.cs

```diff
@@ -21,6 +22,11 @@ public string Login(string username, string password)
             return SessionToken;
         }
 
+        public void Login(string accessToken)
+        {
+            throw new NotImplementedException();
```

Comment:
Not sure how this object is used, but could this just return the `accessToken`, guess that isn;t really helpful.